dcicutils/ff_utils.py

```diff
@@ -1010,21 +1010,51 @@ def remove_keys(my_dict, remove_list):
     return store, list(item_uuids)
 
 
-def get_health_page(key=None, ff_env=None):
-    """
-    Simple function to return the json for a FF health page given keys or
-    ff_env. Will return json containing an error rather than raising an
-    exception if this fails, since this function should tolerate failure
-    """
+def _get_page(page='/health', key=None, ff_env=None):
```

Comment:
This is a pretty darned weird default. Is there some reason not to just require this argument? I think somehow saying `._get_page()` to mean "get the health page" is just odd.

If you're trying to encourage (well, require) the argument to be a keyword argument, you can do:
```
def _get_page(*, page, key=None, ff_env=None):
```